dom/abort/event.any.js

```diff
@@ -0,0 +1,22 @@
+async_test(t => {
```

Comment:
This can be a sync test (and it's good to test that things happen synchronously)